docs/about/team/internships.md

```diff
@@ -4,6 +4,11 @@ sidebar_position: 4
 
 # Internships at the AI Lab
 
+### [Werbrouck, Francois] (https://www.linkedin.com/in/francois-werbrouck-b17296202) - Base de données Hivers 2024
```

Comment:
I think `Hiver` is better then `Hivers`

Reply:
my mistake, I went ahead and fixed it in the minor fix commit